### PR TITLE
Update screenshot tool

### PR DESCRIPTION
**Pull request by maxme**

### CODE REVIEW

File: WordPress/src/androidTest/monkeys/playstore-screenshots.py [hunk 1]
```diff
@@ -21,88 +21,51 @@ def action_login(device, serialno):
...
 
-# Menu drawer should be opened when calling this
-def action_open_reader_posts_i_like(device, serialno):
+def action_open_reader_freshlypressed(device, serialno):
     print("open reader")
     # Open the reader
-    for i in range(3):
-        device.press('KEYCODE_DPAD_UP')
-    device.press('KEYCODE_DPAD_DOWN')
-    device.press('KEYCODE_ENTER')
```
Comment (tonyr59h) [inline WordPress/src/androidTest/monkeys/playstore-screenshots.py]: Removing the ENTER presses means it won't open the tab, just highlight the tab in the tab bar. It all started working when I added the ENTER press back.

Comment (maxme) [inline WordPress/src/androidTest/monkeys/playstore-screenshots.py]: I've just tested and it works without the enter key. I think it's an Android version issue (that works on lollipop). I added the enter key in b52706693a9782904ec3ccffd9dd4dbb008b1037 - it also works on lollipop.

Comment (tonyr59h) [inline WordPress/src/androidTest/monkeys/playstore-screenshots.py]: Ah, sorry I didn't catch that. Only have my rooted S3, wasn't testing on Lollipop.

**Comment by tonyr59h** [comment]
:shipit:

**Pull request merged**